rerun_cpp/src/rerun/archetypes/bar_chart_ext.cpp

```diff
@@ -0,0 +1,126 @@
+#include "bar_chart.hpp"
+
+// #define EDIT_EXTENSION
+
+namespace rerun {
+    namespace archetypes {
+
+#ifdef EDIT_EXTENSION
+        // [CODEGEN COPY TO HEADER START]
+
+        inline BarChart(rerun::datatypes::TensorBuffer buffer) {
+            this->values = rerun::components::TensorData::one_dim(std::move(buffer));
+        }
+
+        // --------------------------------------------------------------------
+        // Implicit constructors:
```

Comment:
a bit redundant to call them "implicit constructors". All constructors are implicit unless they are explicitely marked as explicit